lib/pure/uri.nim

```diff
@@ -166,17 +166,20 @@ iterator decodeQuery*(data: string): tuple[key, value: TaintedString] =
   runnableExamples:
     import std/sugar
     let s = collect(newSeq):
```

Comment:
`let s = collect(newSeq):`
=>
`let s = collect:`

is now the preferred way (implies `newSeq`)